src/main.nf

```diff
@@ -49,15 +48,70 @@ else {
 }
 
 
+if ( params.run_manta ) {
+    process index_bamfile {
+        input:
+            file 'bamfile' from bamfile
+        output:
+            file 'bamfile.bai' into bamfile_index
+
+        module 'bioinfo-tools'
+        module "$params.modules.samtools"
+
+        """
+        samtools index bamfile
+        """
+    }
+
+    process run_manta {
+        input:
+            file 'bamfile_tmp' from bamfile
+            file 'bamfile.bai' from bamfile_index
+        output:
+            file 'sample.manta.sv.vcf.gz' into manta_vcf_gz
+            file 'sample.manta.sv.vcf' into manta_vcf
+            file 'sample.manta.sv.bed' into manta_bed
+
+        publishDir 'results'
+
+        module 'bioinfo-tools'
+        module "$params.modules.manta"
+
+        """
+        # Manta follows symlinks and expects the index to be with the original
+        # file, so we copy it here and then clean up afterwards.
+        # TODO, this is fixed in manta v0.29.5, https://github.com/Illumina/manta/issues/32
+        DIR=`pwd`
+        function cleanup() {
+            cd \$DIR
+            if [ -f bamfile ]; then
+                rm bamfile
+            fi
+        }
+        trap cleanup EXIT
```

Comment:
Maybe it is just me, but I don't know what trap does ... not sure, but should one add a comment to tell what it does?